to-headers/index.js

```diff
@@ -10,7 +10,7 @@
  * @return {Headers|Any}
  */
 module.exports = (headers) => {
-    if (headers && typeof headers === 'object' && headers !== null && !(headers instanceof Headers)) {
+    if (headers && typeof headers === 'object' && headers !== null && supportsHeaders() && !(headers instanceof Headers)) {
```

Comment:
I believe both `typeof headers === 'object'` and `typeof headers !== null` cover the first condition `headers` condition.

Reply:
But typeof null === 'object'

Reply:
Oh, i see. It's falsy.